Packs/CortexXDR/Playbooks/Cortex_XDR_-_Large_Upload.yml

```diff
@@ -0,0 +1,4004 @@
+  playbookInputQuery:
+- key: FurtherInvestigation
+  value:
+    simple: "False"
+  required: true
+  description: |-
+    Determines whether an incident should be further investigated if similar previous false positive incidents were found.
+    Possible values:True/False. Default: False.
+  playbookInputQuery:
+- key: AutoBlockIndicators
+  value:
+    simple: "True"
+  required: true
+  description: |-
+    Determine whether the given indicators be automatically blocked, or if the user should be given the option to choose.
+    Possible values:True/False. Default: True.
+    If set to False - no prompt will appear, and all provided indicators will be blocked automatically.
+    If set to True - the user will be prompted to select which indicators to block.
+  playbookInputQuery:
+- key: BlockIndicators_UserVerification
+  value:
+    simple: "False"
+  required: false
+  description: |-
+    Determine whether the blocking of any indicator requires the verification of the user.
+    Possible values:True/False. Default: False.
+  playbookInputQuery:
+- key: EarlyContainment
+  value:
+    simple: "True"
+  required: true
+  description: |-
+    Whether early containment should be allowed when the IP address is known to be malicious.
+    Possible values:True/False. Default: True.
+  playbookInputQuery:
+- key: AutoIsolateEndpoint
+  value:
+    simple: "False"
+  required: true
+  description: |-
+    Whether to isolate the initiating endpoint automatically if the investigation verdict is malicious.
+    Possible values:True/False. Default: False.
+  playbookInputQuery:
+- key: Alert_ID
+  value:
+    complex:
+      root: PaloAltoNetworksXDR.Incident.alerts
+      filters:
+      - - operator: containsGeneral
+          left:
+            value:
+              simple: PaloAltoNetworksXDR.Incident.alerts.name
+            iscontext: true
+          right:
+            value:
+              simple: Large Upload
+          ignorecase: true
+      accessor: alert_id
+      transformers:
+      - operator: uniq
+  required: false
+  description: The Cortex XDR alert ID.
+  playbookInputQuery:
+- key: Transferred_Data _Threshold
+  value:
+    simple: "150"
+  required: true
+  description: |-
+    Specify the uploaded data threshold volume (in MB) from which large upload alerts should be investigated.
+    By setting a threshold, you will be able to determine which large upload alerts require investigation.
+    Default value: 150 (MB).
+  playbookInputQuery:
+- key: FWApps_Processes_Whitlist
+  value:
+    simple: ip,tcp,udp,ssl,syslog,quic,Chrome.exe,Firefox.exe,Opera.exe,Safari.exe,iexplore.exe,msedge.exe,brave.exe
+  required: false
+  description: A list of known and authorized FW application IDs and processes used in the organization.
+  playbookInputQuery:
+outputs: []
+tests:
+- No tests (auto formatted)
+fromversion: 6.10.0
```

Comment:
@ssokolovich 
The validation test of the build fails since the 'fromversion' of the 'XDR Alerts Handling v2' is set to 6.5.0 while the large upload 'fromversion' is set to 6.10.0.

Should I lower the 'fromversion' of the large upload playbook to 6.5.0?
![Screenshot 2024-03-13 at 21 46 20](https://github.com/demisto/content/assets/112805149/e39012dd-6913-41e6-93cc-e1ba6eb731ef)

Reply:
keep it as it is and add the skip if unavailable in the XDR alerts handling

Reply:
done